Source/OxyPlot/Foundation/CodeGenerator/CodeGenerator.cs

```diff
@@ -137,7 +137,11 @@ public string ToCode()
         private string Add(object obj)
         {
             var type = obj.GetType();
+#if NET40
+            var hasParameterLessCtor = type.GetConstructors().Any(c => c.GetParameters().Length == 0);
```

Comment:
`DeclaredConstructors` retrieves static constructors, which `Type.GetConstructors` does not (without suitable `BindingFlags`). This code probably shouldn't be picking up static constructors at all, so I think the NET4.0 code is fine, but the NET Standard code needs an additional check so it only counts non-static constructors.

Example of counting static constructors:

     typeof(System.Collections.Generic.List<int>).GetTypeInfo().DeclaredConstructors.Count(ci => ci.GetParameters().Length == 0) // returns 2

Reply:
Thanks! It is likely to be other bugs like that...

I think OxyPlot.Tests and ExampleLibrary should also target net40, so we could run all tests on all framework versions...